app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class = form_wrapper>
+<h2 class="sign_up">Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+   <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+    
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio %>
+  </div>
+
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.file_field :photo %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ Required ] I think it would be better if you style the `SignUp` form the same way it is styled in the project wireframe. The Sign-Up button should be the same `width` as the input fields. I think you could also try to align the [Login](), and [Didn't receive confirmation instructions?]() to be in line with the form so that they are not very far from the form. Please see the screenshot below for more details and rectify this.

## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)

## Your design
![Screenshot from 2023-06-22 00-58-51](https://github.com/ernestmusong/Blog-app/assets/16392046/9b75f55a-f9fd-41d8-b0fe-13a7c862a306)